_includes/tools-menu.html

```diff
@@ -4,8 +4,8 @@ <h3>Tools</h3>
 {% for group in groups %}
 <ul class="uk-nav uk-nav-default tm-nav uk-margin-top">
   <li class="uk-nav-header">{{ group.name }}</li>
-  {% for item in group.items %}
-  <li><a href="{{ item.url }}">{{ item.name }}</a></li>
+  {% for item in group.items | sort: "name" %}
```

Comment:
@sammyjava @alancleary 
This appears to be invalid syntax. Setting the [Liquid option](https://jekyllrb.com/docs/configuration/liquid/) `error_mode: strict` in jekyll-starter-legumeinfo's _config.yml to abort on liquid parse errors:
```
liquid:
  error_mode: strict
```
Attempting to build the starter site, updated to use current version of the jekyll-theme-legumeinfo submodule, with `docker compose up` results in the error: 
```
Liquid Exception: Liquid syntax error (/srv/jekyll/_themes/jekyll-theme-legumeinfo/_includes/left-menu.html line 7): Expected end_of_string but found pipe in "item in group.items | sort: "name"" in /srv/jekyll/_themes/jekyll-theme-legumeinfo/_layouts/default.html
```
Without setting the error_mode, the site builds, though the tools menu entries aren't sorted (but that might be a feature if it allows the tools to be explicitly ordered in the tools.yml).

It might be worth setting `error_mode: strict` in jekyll-starter-legumeinfo's _config.yml (and probably `strict_filters: true` as well) to catch such errors when testing the theme.

Reply:
Interesting. I didn't write that code, just copied it over from jekyll-legumeinfo. I expect @alancleary will happily clean it up. I agree that strict errors is probably a good idea at least on the starter site since we're supposed to use that for testing the theme.

Reply:
Good catch @nathanweeks. I think having the tools ordered as they appear in `tools.yml` is preferable since the ordering by name this code is supposed to achieve is still facilitated by that approach. Does one of you mind making a PR that fixes this? I will happily take all the help I can get maintaining our codebase ;)

Reply:
I'll bounce it to @nathanweeks to also put the `error_mode: strict` config option in and give 'er a test.